migration/src/main/java/uk/ac/bbsrc/tgac/miso/migration/destination/MisoServiceManager.java

```diff
@@ -1324,6 +1328,7 @@ public void setDefaultTissueMaterialDao() {
 
   private void updateTissueMaterialDaoDependencies() {
     if (sampleService != null) sampleService.setTissueMaterialDao(tissueMaterialDao);
+    updateSamplePurposeDaoDependencies();
```

Comment:
Wrong place. This should be done in the setSamplePurposeDao method, where it was originally